src/components/Navbar/SettingsDrawer/WordTooltipSection.module.scss.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import useTranslation from 'next-translate/useTranslation';
+import { useDispatch, useSelector } from 'react-redux';
+
+import Section from './Section';
+import styles from './WordByWordSection.module.scss';
+
+import Checkbox from 'src/components/dls/Forms/Checkbox/Checkbox';
+import HelperTooltip from 'src/components/dls/HelperTooltip/HelperTooltip';
+import {
+  setShowTooltipFor,
+  selectShowTooltipFor,
+} from 'src/redux/slices/QuranReader/readingPreferences';
+import { areArraysEqual } from 'src/utils/array';
+import { WordByWordType } from 'types/QuranReader';
+
+const WordTooltipSection = () => {
+  const { t } = useTranslation('common');
+  const dispatch = useDispatch();
+  const showTooltipFor = useSelector(selectShowTooltipFor, areArraysEqual);
+
+  return (
+    <Section>
+      <Section.Title>
+        {t('word-tooltip')}
+        <HelperTooltip>{t('settings.wbw-tooltip')}</HelperTooltip>
+      </Section.Title>
+      <Section.Row>
+        <div className={styles.checkboxContainer}>
+          <div>
+            <Checkbox
+              checked={showTooltipFor.includes(WordByWordType.Translation)}
+              id="word-tooltip-translation"
+              name="word-tooltip-translation"
+              label="Translation"
+              onChange={(checked) => {
+                const nexShowTooltipFortValue = new Set(showTooltipFor);
```

Comment:
What is the need of using a `Set` here? also I think we should have this as a separate function instead of an inline prop